devito/profiling.py

```diff
@@ -100,7 +100,7 @@ class AdvancedProfiler(Profiler):
     # Override basic summary so that arguments other than runtime are computed.
     def summary(self, arguments, dtype):
         """
```

Comment:
Could you **cut** this entire  docstring and paste it in the superclass Profiler.summary (deleting what's currently in there)?

Reply:
ok